src/Equinor.ProCoSys.IPO.Command/InvitationCommands/EditInvitation/EditInvitationCommandHandler.cs

```diff
@@ -115,13 +115,13 @@ private async Task<List<McPkg>> GetMcPkgScopeAsync(IList<string> mcPkgNos, strin
                     throw new IpoValidationException("Could not find all mc pkgs in scope.");
                 }
 
-                var initialMcPkg = mcPkgsFromMain.FirstOrDefault();
-                if (initialMcPkg != null)
+                var initialCommPkg = mcPkgsFromMain.FirstOrDefault();
```

Comment:
why this rename, for me initialMcPkg is better since picking the first from mcPkgDetailsList

Reply:
sorry just a copypaste mistake